kube/README.md

```diff
@@ -0,0 +1,74 @@
+# Run Airbyte Locally from Kubernetes
+
+All commands should be run from the root Airbyte source directory.
+
+1. Make sure you are using the correct Kubernetes context with `kubectl config current-context`
+1. Apply the manifests for one of:
+    * Latest stable version
+        1. Apply with `kubectl apply -k kube/overlays/stable`
+    * If you want to use the latest development version:
+        1. Build the `dev` version of images with `./gradlew composeBuild`
+        1. Apply with `kubectl apply -k kube/overlays/dev`
+1. Wait for pods to be "Running" on `kubectl get pods | grep airbyte`
+1. Run `kubectl port-forward svc/airbyte-server-svc 8001:8001` in a new terminal window.
+    * This exposes `airbyte-server`, the Airbyte api server.
+    * If you redeploy `airbyte-server`, you will need to re-run this process.
+1. Run `kubectl port-forward svc/airbyte-webapp-svc 8000:80` in a new terminal window.
+    * This exposes `airbyte-webapp`, the server for the static web app.
+    * These static assets will make calls to the Airbyte api server, which is why both services needed to be port forwarded.
+    * If you redeploy `airbyte-webapp`, you will need to re-run this process.
+1. Go to http://localhost:8000/ and use Airbyte!
+
+## Operating Airbyte on Kubernetes
+
+### Kustomize
+
+We use [Kustomize](https://kustomize.io/), which is built into `kubectl` to allow overrides for different environments.
+
+Our shared resources are in the `kube/resources` directory, and we define overlays for each environment. We recommend creating your own overlay if you want to customize your deployments. 
+
+Example `kustomization.yaml` file:
+```
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - https://airbytehq/airbyte.git//kube/overlays/stable?ref=master
```

Comment:
is the extra `/` between .git and kube intentional?

Reply:
yeah

Reply:
Just tested, it works without it. Somehow we all missed this didn't have github.com though xD